### DISCUSSION TITLE
what to do if im no more joined on some of the servers?

### DISCUSSION THREAD

**Discussion by azamet90** [victornpb/undiscord#466] [1. Questions & Answers]
im using the import json option but many of the servers im no more in and i dont even remember which server they are. i think it is not removing all of them because of im not joined on the servers so how to join to them automatic?

**Comment by victornpb** [comment]
I’m not sure if that's possible. because you need invites to join servers, and invites are not linked to server ids

**Reply by victornpb** [reply]
Checkout this thread by the community https://github.com/victornpb/undiscord/discussions/429
They are discussing how to get discord to comply with privacy laws
